packages/connect/src/core/index.ts

```diff
@@ -546,6 +558,17 @@ const onCall = async (context: CoreContext, message: IFrameCallMessage) => {
             await waitForPopup(context);
             // show message about transport
             sendCoreMessage(createUiMessage(UI.TRANSPORT));
+
+            // Retry onCallDevice again
+            // NOTE: this should change after multi-transports refactor, where transport will be always alive
```

Comment:
I believe that some kind of retry-logic will always make sense. It can always be possible that none transport will be available. And it will always be useful to be able to pause, solve the problem, come back and continue